src/js/components/container/NewFoodFormComponent.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Form } from 'reactstrap';
+import FormInput from "../presentational/FormInputComponent.jsx";
+
+class NewFoodForm extends React.Component {
+    constructor() {
+      super();
+      this.state = {
+        title: ""
```

Comment:
Is this used anywhere?

Reply:
You know, I think this might have been a typo or something. I don't see it anymore. What I have there now is 
```suggestion
       constructor(props) {
    super(props);
    this.state = {
      formControls: {
        name: "",
        type: "",
        region: "",
        rating: ""
      }
    };
```